include/vicon_calibration/camera_models/CameraModel.h

```diff
@@ -189,6 +200,8 @@ class CameraModel {
   uint32_t image_height_{0};
   uint32_t image_width_{0};
   std::vector<double> intrinsics_;
+  float fov{0};
```

Comment:
we use double everywhere else, so make this a double to be consistent

Reply:
Yes